src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1368,20 +1368,54 @@ private void VisitStatements(ImmutableArray<IOperation> statements)
         {
             for (int i = 0; i < statements.Length; i++)
             {
-                if (statements[i] is VariableDeclarationGroupOperation declarationGroup
-                    && declarationGroup.DeclarationKind != VariableDeclarationKind.Default)
+                VisitPossibleUsingDeclaration(statements[i], statements, i, out var visitedUsingDeclaration);
```

Comment:
This might be better expressed as a `TryVisitUsingDeclaration` that returns a `bool`, rather than having an out variable.